json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -404,9 +404,6 @@ protected function render_response_key( $key, &$response, $is_user_logged_in ) {
 			case 'quota' :
 				$response[ $key ] = $this->site->get_quota();
 				break;
-			case 'launch_status' :
```

Comment:
It looks like this only got moved up a little bit in the wpcom diff, but here it's removed altogether.